neurom/fst/__init__.py

```diff
@@ -137,24 +137,47 @@ def get(feature, obj, **kwargs):
     '''Obtain a feature from a set of morphology objects
 
     Parameters:
-        feature (string): feature to extract.
-        obj: a neuron, population or neurite tree.
-        **kwargs: parameters to forward to underlying worker functions.
+        feature(string): feature to extract
+        obj: a neuron, population or neurite tree
+        **kwargs: parameters to forward to underlying worker functions
 
     Returns:
         features as a 1D or 2D numpy array.
-        '''
+
+    '''
 
     feature = (NEURITEFEATURES[feature] if feature in NEURITEFEATURES
                else NEURONFEATURES[feature])
 
     return _np.array(feature(obj, **kwargs))
 
+_INDENT = ' ' * 4
+
+
+def _indent(string, count):
+    '''indent `string` by `count` * INDENT'''
+    indent = _INDENT * count
+    ret = indent + string.replace('\n', '\n' + indent)
+    return ret.rstrip()
+
+
+def _get_doc():
+    '''Get a description of all the known available features'''
+    def get_docstring(func):
+        '''extract doctstring, if possible'''
+        docstring = ':\n'
+        if not isinstance(func, partial) and func.__doc__:
```

Comment:
I believe I'm going to have to create a FeatureFunc structure so that I can override the docstrings for lambda's and partials, since many of the features use this, but it means their description is blank.

I will probably do that in another commit, as this one is big enough as is, I think